doc/en/user/source/services/wfs/webadmin.rst

```diff
@@ -66,6 +66,17 @@ A list of values to be declared in all feature types can be provided in the WFS
 
 The list will be used only for the capabilities document generation. It does not limit the actual target SRS usage in GetFeature requests.
 
+Configuration of Output Format types allowed in GetFeature
```

Comment:
Capitalization of section names:

> Configuration of output format types allowed in GetFeature

- https://docs.geoserver.org/latest/en/docguide/style.html#capitalization-of-page-names